xapian-core/weight/Makefile.mk

```diff
@@ -2,6 +2,10 @@ noinst_HEADERS +=\
 	weight/weightinternal.h
 
 EXTRA_DIST +=\
+	weight/collate-wdf-norm\
+	weight/collate-idf-norm\
+	weight/wdf-norm-dispatch.h\
+	weight/idf-norm-dispatch.h\
```

Comment:
It'd be better to add these in sorted order - that way it's easier to quickly locate an entry, and that also makes it harder to accidentally add a duplicate of an entry which is already there (because you'd be trying to add it in the place it already is).

Reply:
Updated.